platform/ios/app/MBXAnnotationView.m

```diff
@@ -49,17 +49,4 @@ - (void)setDragState:(MGLAnnotationViewDragState)dragState animated:(BOOL)animat
 
 }
 
-- (nullable id<CAAction>)actionForLayer:(CALayer *)layer forKey:(NSString *)event
```

Comment:
Just noting that I think removing this makes sense only because I also think that we should feature UIView animation blocks as the recommended way of performing annotation view animations. Animation blocks have the convenient side effect of animating any open callout view along with the annotation view. cc @1ec5